miniapps/tools/display-basis.cpp

```diff
@@ -72,64 +73,71 @@ struct DeformationData
 class Deformation : public VectorCoefficient
 {
 public:
+   enum DefType
+   {
+      INVALID,
+      UNIFORM,
+      SQUEEZE,
+      SHEAR
+   };
 
-   enum DefType {INVALID, UNIFORM, SQUEEZE, SHEAR};
-
-   Deformation(int dim, DefType dType, const DeformationData & data)
+   Deformation(int dim, DefType dType, const DeformationData &data)
       : VectorCoefficient(dim), dim_(dim), dType_(dType), data_(data) {}
 
    void Eval(Vector &v, ElementTransformation &T, const IntegrationPoint &ip);
    using VectorCoefficient::Eval;
+
 private:
-   void Def1D(const Vector & u, Vector & v);
-   void Def2D(const Vector & u, Vector & v);
-   void Def3D(const Vector & u, Vector & v);
+   void Def1D(const Vector &u, Vector &v);
+   void Def2D(const Vector &u, Vector &v);
+   void Def3D(const Vector &u, Vector &v);
 
-   int     dim_;
+   int dim_;
    DefType dType_;
-   const DeformationData & data_;
+   const DeformationData &data_;
 };
 
-string   elemTypeStr(const Element::Type & eType);
-inline bool elemIs1D(const Element::Type & eType);
-inline bool elemIs2D(const Element::Type & eType);
-inline bool elemIs3D(const Element::Type & eType);
+string elemTypeStr(const Element::Type &eType);
+inline bool elemIs1D(const Element::Type &eType);
+inline bool elemIs2D(const Element::Type &eType);
+inline bool elemIs3D(const Element::Type &eType);
```

Comment:
There are a lot of styling changes in this file that make it difficult to review. Is it possible to revert and introduce just the changes needed for serendipity support?

Reply:
I think all the non-essential changes in the display-basis.cpp file are a result of running `astyle` on the version of the file that is already part of master.  I'm not familiar with preferred practices in this regard so just let me know the best way to proceed.

Reply:
This must have been with the different version of `astyle` (note that we conform to [version 2.05.1](https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#developer-guidelines)).

Can you introduce the changes in the version from master and commit that?

Reply:
The correct version of astyle can be found in the tpls repo.

Reply:
Looks much better now, thank you